### ISSUE TITLE
The result of the official code

### ISSUE THREAD

**Issue by 1962975362** [ZJZAC/Deep-Model-Watermarking#3]
I would like to ask whether the training results of the open source code can meet the indicators in the paper. Why did my metrics fluctuate so much during my training. Could you please provide the pre-training model and the schematic diagram of training convergence? Thank you very much

**Comment by ZJZAC** [comment]
Thanks for your reaching out. 
I am sorry for my unstable access  to GitHub recently.  You can direclty email me attached with your results including the metric you use, and we can discuss about them.
